git_machete/cmd.py

```diff
@@ -1979,7 +1999,12 @@ def parse_direction(b, down_pick_mode):
             read_definition_file()
             slide_out(params or [current_branch()])
         elif cmd in ("s", "status"):
-            expect_no_param(parse_options(args, "l", ["list-commits"]))
+            expect_no_param(parse_options(args, "l", ["list-commits", "color="]))
+            if opt_color not in ("always", "auto", "never"):
+                raise MacheteException("Invalid argument for: --color. Valid arguments: always, auto, never.")
+            else:
+                global ascii_only
+                ascii_only = False if opt_color == "always" or (opt_color == "auto" and sys.stdout.isatty()) else True
```

Comment:
Or just `ascii_only = opt_color == "always" or (opt_color == "auto" and sys.stdout.isatty())`